CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+
+* Demonstrating empathy and kindness toward other people.
+* Being respectful of differing opinions, viewpoints, and experiences.
+* Giving and gracefully accepting constructive feedback.
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience.
+
+As participants in the PureScript community, we pledge to make participation in our community a harassment-free experience for everyone -- regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
+
+## What is not okay?
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of any kind.
+* Trolling, insulting or derogatory comments, and personal or political attacks.
+* Public or private harassment.
+* Publishing others’ private information, such as a physical or email address, without their explicit permission.
+* Other conduct which could reasonably be considered inappropriate in a professional setting.
+
+
+## Where does this Code of Conduct apply?
+
+This code of conduct covers anyone participating in PureScript community channels, including our Discord server, Discourse forum, and GitHub properties. For members of the PureScript team, it applies anywhere they are representing the PureScript project (for example, on social media).
+
+## How do members report violations?
+
+You can report abusive, harassing, or otherwise unacceptable behavior to the community moderators team or to a particular moderator if you are not comfortable sharing your concern with the full team.
+
+All complaints will be reviewed and investigated promptly and fairly. Your communication with moderators will not be shared with anyone else without your permission.
+
+Here are per-platform instructions:
+
+* Discord: Send a private message to the `@Moderators` team. To find a specific moderator to message, use the members sidebar under the `Moderators` role to find their profile.
+* Discourse: Send a private message to the `moderators` group. To find a specific moderator to message, see the `moderators` group page: https://discourse.purescript.org/g/moderators
+* All Other: If an event occurs somewhere else, please reach out to moderators on either Discord or Discourse.
+
+## How will you enforce the rules?
+
+Our community leaders follow the below guidelines to determine the consequences for any action they deem in violation of this Code of Conduct:
+
+1. **Correction**
+
+   Cause: Use of inappropriate language or other behavior deemed unprofessional or unwelcome in the community.
+
+   Consequence: A private, written warning from community leaders, providing clarity around the nature of the violation and an explanation of why the behavior was inappropriate. A public apology may be requested.
```

Comment:
We may also want to include that moderators can edit or delete a message, or ask that the person edits or deletes their message, if it's deemed something that shouldn't be preserved in the channel. This is a sensitive line to walk, but I suspect it will sometimes be necessary.

Reply:
I agree. Also, replace "community leaders" with "the moderators", remove the comma after leaders, replace "providing clarity around" with "clarifying", replace "an explanation of" with "explaining". 

So it now reads "A private, written warning from moderators clarifying the nature of the violation and explaining why the behavior was inappropriate."